### PR TITLE
Use GHA instead of Travis

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Review by findepi — APPROVED** [review]

**Comment by losipiuk** [comment]
@findepi should we change some repository configuration so those github checks actually run here. The checks were executed on source branch for this PR in my fork, but not on the PR itself.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: ci
+
+on:
+  - push
+  - pull_request
+
+
```
Comment (electrum) [inline .github/workflows/ci.yml:7]: Remove double blank line

**Review by electrum — APPROVED** [review]

**Comment by findepi** [comment]
Actions are enabled in the repo settings. @electrum is it that the actions yml needs to be merged into master before actions are active?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (findepi) [reply .github/workflows/ci.yml:7]: Sorry, i overlooked this.

**Pull request merged**